common/src/main/java/com/datatorrent/common/util/FSStorageAgent.java

```diff
@@ -71,12 +71,10 @@ public FSStorageAgent(String path, Configuration conf)
 
```

Comment:
This file (FSStorageAgent.java)  only contains formatting changes. Can you revert it back.